<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMediaPlayer</class>
 <widget class="QWidget" name="QtMediaPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtMediaPlayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	min-width:32px;
	max-width:32px;
	min-height:32px;
	max-height:32px;

	border:none;
}

QPushButton:hover
{
	background-color:rgb(160, 160, 160)
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="MusicList">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="CurDur">
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="SplitCh">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Dur">
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="PlaySlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="OpenDirBtn">
        <property name="toolTip">
         <string>打开文件夹</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/Resource/OpenDir.png</normaloff>:/Resource/OpenDir.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PrevBtn">
        <property name="toolTip">
         <string>上一曲</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/Resource/Prev.png</normaloff>:/Resource/Prev.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PlayBtn">
        <property name="toolTip">
         <string>播放/暂停</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/Resource/PlayOrPause.png</normaloff>:/Resource/PlayOrPause.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="NextBtn">
        <property name="toolTip">
         <string>下一曲</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/Resource/Next.png</normaloff>:/Resource/Next.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Volume">
        <property name="toolTip">
         <string>音量</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/Resource/Volume.png</normaloff>:/Resource/Volume.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="VolumeSlider">
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
